rpc/v1/openapi.yaml

```diff
@@ -0,0 +1,98 @@
+openapi: 3.0.3
+servers:
+  - url: 'http://localhost:50832'
+info:
+  description: >-
+    This is the API definition Pocket Network core RPC call.
+
+
+    Pocket is a distributed network that relays data requests and responses to and from any blockchain system.
+
+
+    Pocket verifies all relayed data and proportionally rewards the participating nodes with native cryptographic tokens.
+  version: 1.0.0
+  title: Pocket Network
+  termsOfService: 'https://pokt.network/terms/'
+  contact:
+    email: hola@pokt.network
+  license:
+    name: MIT License
+    url: 'https://github.com/pokt-network/pocket/blob/main/LICENSE'
+tags:
+  - name: version
+    description: Version of the Pocket API
+  - name: health
+    description: Liveness of the Pocket API node
+  - name: client
+    description: Dispatch and relay services
+paths:
+  /v1/health:
+    get:
+      tags:
+        - health
+      summary: Get the liveness of the Pocket API node
+      responses:
+        '200':
+          description: Healthy
+        '404':
+          description: Unhealthy - Unreachable
+        '500':
+          description: Unhealthy - Server Error
+  /v1/version:
+    get:
+      tags:
+        - version
+      summary: Get the current version of the Pocket Network API
+      responses:
+        default:
+          description: Default response
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: 1.0.0
+  /v1/client/broadcast_tx_sync:
+    post:
+      tags:
+        - client
+      summary: Relays a raw transaction
```

Comment:
Is this necessary related to Pocket relays or is it any transaction (e.g. broadcasting the transmission of funds)?

To avoid conflating terminology with Pocket's "relays", maybe change the wording here to "Broadcast raw transaction bytes"?